docs/content/how-to/disaster-recovery/backup-and-restore-oadp-1-5.md

```
@@ -41,6 +42,26 @@ oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=cr
 
     If using AWS S3, additional AWS resources must be created to enable data backup and restoration. Follow [these instructions](https://docs.openshift.com/container-platform/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#migration-configuring-aws-s3_installing-oadp-aws) to set up the necessary configurations.
 
+For Azure Blob Storage, create the credentials in the following format:
+
+```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)